Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -87,72 +94,95 @@ def get_issue_info(cls, args, repository, **kwargs):
             if not issue:
                 sys.stderr.write('Failed to create new issue\n')
                 return None
+            args.issue = issue.link
             print("Created '{}'".format(issue))
         elif not Tracker.instance():
             sys.stderr.write('Could not find tracker instance.\n')
+
+        rdar = Branch.create_radar(args, repository, issue)
+        if rdar:
+            log.info("Created {}".format(rdar))
+
         return issue
 
     @classmethod
     def create_revert_commit_msg(cls, args, repository, **kwargs):
-        # Make sure we have the commit that user want to revert
-        try:
-            commit = repository.find(args.commit, include_log=True)
-        except (local.Scm.Exception, ValueError) as exception:
-            # ValueErrors and Scm exceptions usually contain enough information to be displayed
-            # to the user as an error
-            sys.stderr.write('Could not find "{}"'.format(args.commit) + '\n')
-            return 1
-        bug_urls = []
-        commit_title = None
-        for line in commit.message.splitlines():
-            if not commit_title:
-                commit_title = line
-            tracker = Tracker.from_string(line)
-            if tracker:
-                bug_urls.append(line)
+        commit_identifiers = []
+        commits_to_revert = []
+        reverted_changeset = ''
+        # Retrieve information for commits to be reverted.
+        for c in args.commit:
+            try:
+                commit = repository.find(c, include_log=True)
+            except (local.Scm.Exception, ValueError) as exception:
+                # ValueErrors and Scm exceptions usually contain enough information to be displayed
+                # to the user as an error
+                sys.stderr.write('Could not find "{}"'.format(c) + '\n')
+                return None, None
+            commits_to_revert.append(commit.hash)
+            bug_urls = []
+            commit_title = None
+            for line in commit.message.splitlines():
```

Comment:
Can we just use `commit.issues` here? Notably, this won't work in repositories which do `<bug> [title]`

Reply:
Test comment

Reply:
Test comment B